py_rete/production.py

```diff
@@ -116,10 +129,9 @@ def activations(self):
     def get_rete_conds(self):
         if self.pattern is None:
             return ([],)
-        disjuncts = compile_disjuncts(self.pattern)
-        return [list(get_rete_conds(AND(*disjunct)))
-                if isinstance(disjunct, tuple) else
-                list(get_rete_conds(AND(disjunct))) for disjunct in disjuncts]
```

Comment:
This condition handled some special case that the disjunctive normal form conversion previously didn't -- now it can be dropped, and the return form is always the same, i.e. a list.